src/config.js

```diff
@@ -0,0 +1,33 @@
+define(['lodash-amd/modern/objects/defaults',], function (defaults) {
+
+  var defaultOptions = {
+    allowBlockElements: true,
+    debug: false,
+    undo: {
+      manager: false,
+      enabled: true,
+      limit: 100,
+      interval: 250
+    },
+    defaultCommandPatches: [
+      'bold',
+      'indent',
+      'insertHTML',
+      'insertList',
+      'outdent',
+      'createLink'
+    ]
+  };
+
+
+  function checkOptions(userSuppliedOptions) {
+    var options = userSuppliedOptions || {};
+
+    return Object.freeze(defaults(options, defaultOptions));
```

Comment:
That's a good point. I think it will. But @rrees - are we retiring FF31/32 support?